app/bundles/DashboardBundle/Resources/views/Widget/detail.html.twig

```diff
@@ -15,22 +24,10 @@
                             <i class="ri-edit-line"></i> Edit
                         </a>
                     </li>
-                    <li role="separator" class="divider"></li>
-                    <li  class="dropdown-header">
-                        {{ 'mautic.dashboard.widget.load.time'|trans({'%time%': (widget.loadTime * 1000)|round(1)}) }}
```

Comment:
Shouldn't we also remove the translation in the ini file?

Reply:
Done!